Dockerfile

```diff
@@ -35,14 +28,14 @@ ENV LANG="en_US.UTF-8" \
 
 # installs system dependencies (required to setup all the tools)
 RUN apt install -qq --yes --no-install-recommends \
-    sudo python-pip python-setuptools file
+    sudo python3-pip python3-setuptools file
 
 # https://buildozer.readthedocs.io/en/latest/installation.html#android-on-ubuntu-18-04-64bit
 RUN dpkg --add-architecture i386 && apt update -qq > /dev/null && \
 	apt install -qq --yes --no-install-recommends \
 	build-essential ccache git libncurses5:i386 libstdc++6:i386 libgtk2.0-0:i386 \
-	libpangox-1.0-0:i386 libpangoxft-1.0-0:i386 libidn11:i386 python2.7 \
-	python2.7-dev openjdk-8-jdk unzip zlib1g-dev zlib1g:i386
+	libpangox-1.0-0:i386 libpangoxft-1.0-0:i386 libidn11:i386 python2.7 python3 \
+	python3-dev openjdk-8-jdk unzip zlib1g-dev zlib1g:i386
```

Comment:
i didn't even install the i386 architecture in any of my recent build, i think this is completly irrelevant now, but i might be missing something, anyway, that can probably be part of another change.

Reply:
Good point, it may be worth to look at in subsequent pull requests.
I actually want to discuss some approaches for Docker